data_structures/binary_tree/basic_binary_tree.py

```diff
@@ -4,13 +4,13 @@ class Node:
     and left, right pointers.
     """
 
-    def __init__(self, data):
-        self.data = data
-        self.left = None
-        self.right = None
+    def __init__(self, data: int) -> None:
+        self.data: int = data
+        self.left: Node = None
+        self.right: Node = None
```

Comment:
Given that None is not a Node, I believe that you would need
https://docs.python.org/3/library/typing.html#typing.Optional